packages/cli/src/lifecycles/context.js

```diff
@@ -16,6 +17,8 @@ module.exports = initContexts = async({ config }) => {
       const userPageTemplate = path.join(userTemplatesDir, 'page-template.js');
       const indexPageTemplate = 'index.html';
       const notFoundPageTemplate = '404.html';
+      const webpackProd = 'webpack.config.prod.js';
```

Comment:
Per my comment here, would it make more sense to pick this up as **webpack**'s default support for _webpack.config.js_?

Or maybe it doesn't matter since we do it programmatically.  Also, I think **webpack** [recommends breaking it up](https://webpack.js.org/guides/production/) actually, so maybe this is fine as is.

Reply:
we do it programmatically, it won't work that way.